<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQCombobox</class>
 <widget class="QWidget" name="MyQCombobox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyQCombobox</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Qt Designer 添加数据项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string> 中国</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>日本</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>朝鲜</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>代码添加数据项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox2"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>获取当前的Index、内容</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
